File: server/tests/fixtures/alpha_rule_triggers/python/naming_project_term_inconsistency.py
# Should trigger: naming.project_term_inconsistency
# Rule detects inconsistent verb usage for same noun phrase (e.g., get_user vs fetch_user)

class UserService:
    def get_user(self, id):  # Uses "get" verb
        pass
    
    def fetch_user(self, id):  # Uses "fetch" verb - inconsistent with get_user!
        pass
    
    def load_user(self, id):  # Uses "load" verb - inconsistent with get_user!
        pass

class OrderManager:
    def create_order(self, data):
        pass
    
    def make_order(self, data):  # "make" is synonym for "create" - inconsistent!
        pass

File: server/tests/fixtures/alpha_rule_triggers/java/naming_project_term_inconsistency.java
// Should trigger: naming.project_term_inconsistency
// Rule detects inconsistent verb usage for same noun phrase (e.g., getUser vs fetchUser)

public class UserService {
    public void getUser(int id) {}  // Uses "get" verb
    
    public void fetchUser(int id) {}  // Uses "fetch" verb - inconsistent!
    
    public void loadUser(int id) {}  // Uses "load" verb - inconsistent!
}

class OrderManager {
    public void createOrder(Object data) {}
    
    public void makeOrder(Object data) {}  // "make" is synonym for "create" - inconsistent!
}
